.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 60
+          days-before-pr-stale: 60
+          days-before-close: 14
+          stale-issue-message: 'This issue has not had any updates in 60 days. If there are no updates/comments within 14 days, this issue will be closed.'
+          stale-pr-message: 'This pull request has not had any updates in 60 days. If there are no updates/comments within 14 days, this pull request will be closed.'
+          close-issue-message: 'This issue has been closed due to inactivity. If your issue has not been resolved, please open a new issue and link this issue.'
+          close-pr-message: 'This issue has been closed due to inactivity. Please open a new pull request if needed.'
+          stale-issue-label: 'Stale'
+          close-issue-label: 'Auto-Closed'
+          stale-pr-label: 'Stale'
+          close-pr-label: 'Auto-Closed'
+          exempt-issue-labels: 'Browser Bug,Bug,dependencies,Documentation,feature-request,Triaged'
```

Comment:
@michhyun1 Can we make the `audio` `video` labels exempt as well or any label associated with other teams? We do not want to auto close issues that might belong to different team.